README.md

```diff
@@ -47,9 +47,9 @@ Files matching this pattern will be included in the manifest.
 
 _Default:_ `"**/*.{js,css,png,gif,ico,jpg,map,xml,txt,svg,swf,eot,ttf,woff,woff2}"`
 
-### ignoreFilePattern
+### fileIgnorePattern
 
-Files matching this pattern will _not_ be included in the manifest even if they match filePattern.
+Files matching this pattern will _not_ be included in the manifest even if they match filePattern. And thus will be allways deployed.
```

Comment:
I agree partially. Then the following docs I think is misleading

> When our deployment plugin is ready to deploy, it retrieves the old manifest (from S3 or wherever its stored), diffs it with the current one, and determines it only has to upload myapp-2.js. For large asset files, this can save alot of time and bandwidth.

It does not diff against new manifest file but it diffs against files that other plugin has been selected. And diffing itself is not concern of this plugin. Can you clarify this matter in docs?